Formula/vowpal-wabbit.rb

```diff
@@ -22,6 +22,7 @@ class VowpalWabbit < Formula
   depends_on "eigen"
   depends_on "fmt"
   uses_from_macos "zlib"
+  patch :DATA
```

Comment:
Yes, we have the same change being made here: https://github.com/VowpalWabbit/vowpal_wabbit/pull/4275

Reply:
Could we wait until that's merged and use the upstream commit then?

Reply:
We are releasing our official 9.6.0 version without the change, as it only seems to affect building through Homebrew. To keep the code itself consistent across all platforms, we'd prefer to use the patch for Homebrew here.

Reply:
Only patching Homebrew is okay, I'm just requesting you use the bellow format for the patch instead of `patch DATA`:
```rb
patch do
  url "the url to the merged upstream commit in patch format"
  sha256 ""
end
```

Reply:
The patch has been updated now.